inprocess/src/main/java/io/grpc/inprocess/InProcessChannelBuilder.java

```diff
@@ -117,10 +117,6 @@ public ClientTransportFactory buildClientTransportFactory() {
     managedChannelImplBuilder.setStatsRecordStartedRpcs(false);
     managedChannelImplBuilder.setStatsRecordFinishedRpcs(false);
     managedChannelImplBuilder.setStatsRecordRetryMetrics(false);
-
-    // By default, In-process transport should not be retriable as that leaks memory.  Since
-    // there is no wire, bytes aren't calculated so buffer limit isn't respected
-    managedChannelImplBuilder.disableRetry();
```

Comment:
In the issue there was a [comment](https://github.com/grpc/grpc-java/issues/8712#issuecomment-976032497) saying retry is enabled by default since 1.40. Since disableRetry() was called here, can you check how that could be the case?

Reply:
On 7/19/22 I added these lines that are being deleted here.  It was a workaround so that ManagedChannelImplBuilder could continue to have retry enabled without InProcessTransport running into the memory leak problem.